docs/InstallGuide.md

```diff
@@ -92,18 +81,59 @@ Modify the following section in /etc/openvnet/webapi.conf on the webapi node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
+        public ""
         port 9101
       }
     }
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the webapi process is run on the same node as vnmgr or vna, specify a different value from vnmgr and vna.
+- **host** : The private IP address of the webapi node.
+- **port** : The port number for the ZeroMQ connection. If the webapi process is run on the same node as vnmgr or vna, specify a different value from vnmgr and vna.
+- **public** : The public IP address if exist.
+
+Datapath is one of the Linux kernel capabilities behaving similar to the Linux bridge.
+Save the following net-script in `/etc/sysconfig/network-scripts` directory as `ifcfg-br0`. However you need to pay attention to several parameters.
+
+* IPADDRESS, NETMASK
+
+The IP address and the netmask of the datapath. Omit them unless IP address is assigned to the datapath.
```

Comment:
How does one know if an IP address is assigned to the datapath? Why is the value 172.16.20.41?